pkg/charts/charts.go

```diff
@@ -16,6 +16,29 @@ import (
 
 const versionPlaceholder = "linkerdVersionValue"
 
+var (
+	// L5dPartials is the list of templates in partials chart
+	// Keep this slice synced with the contents of /charts/partials
+	L5dPartials = []string{
```

Comment:
Any specific reason we have this instead of something that lists the files under `charts/partials/templates/`? It just seems like a possibly "gotcha" for keeping this updated when we could have something that just lists files under this directory.

Reply:
Hmm, Interesting question. We have similar lists for **all charts**, hence I did the same here. 
I think the reason is that we may not want to be dependent on the FS for this, as it can be a real or Virtual, and in the Virtual FS case, I don't see relevant methods do that (or probably I missed something)

@alpeb WDYT?

Reply:
Yeah, `static.Templates` is an `http.FileSystem` that only inplements `Open()`. We could implement a smart `ls` function that in the case of a vfs, it would leverage the fact that in this case `static.Templates` is a function that returns a map whose keys are the file paths... I dunno, could be fun? (if so, certainly in a separate PR)

Reply:
Ah okay I missed that we had this for all charts already. I thought it was just for partials, but I think if its a change we want a separate PR makes sense as well.